app/controllers/cards_controller.rb

```diff
@@ -1,5 +1,50 @@
 class CardsController < ApplicationController
-    def index
-        @cards = Card.all
+  before_filter :find_card,
+      :only => [:show, :update, :edit, :destroy]
+
+  def index
+    @cards = Card.all
+  end
+
+  def create
+    @card = Card.new(card_params)
+    @card.review_date = DateTime.now.next_day(3)
+    if @card.save
+      redirect_to @card
+    else
+      render 'new'
+    end
+  end
+
+  def update
+    if @card.update(card_params)
+      redirect_to @card
+    else
+      render 'edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.